README.md

```diff
@@ -247,6 +247,159 @@ python generate_tags.py --source-path C:/tagging_scripts/tracks --destination-pa
 python tags_to_csv.py --tags-path C:/tagging_scripts/tags --tags-csv C:/tagging_scripts --tags-types "genre" "genre secondary" "mood" "mood secondary" "energy" "vocal presence" "instrument"
 ```
 
+### Java sample
+
+`samples/tagging/java`
+
+#### Set up Maven project
+
+The Java sample shown here uses `Maven` as the Java project management tool. Hence, a `pom.xml` file is required for installing dependencies necessary to run the code samples. A detailed instruction on how to create a Maven project can be found here: https://spring.io/guides/gs/maven/
+
+Initialize your Maven project, in this example, we will use the following project configuration:
+
+```
+<groupId>com.example.tagging</groupId>
+<artifactId>tagging</artifactId>
+```
+
+Your project structure will look something like this:
+
+```
+(/samples/tagging/java/)
+- tagging/
+	| - src/
+  |   | - main/com/example/tagging/
+  |   | - test/
+  | - target/
+  |	- pom.xml
+```
+
+Open `pom.xml` and add the following dependencies in the `<dependencies></dependencies>` tag
+
+```xml
+<!-- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-web -->
+<dependency>
+  <groupId>org.springframework.boot</groupId>
+  <artifactId>spring-boot-starter-web</artifactId>
+  <version>2.3.3.RELEASE</version>
+</dependency>
+<!-- https://mvnrepository.com/artifact/com.github.cliftonlabs/json-simple -->
+<dependency>
```

Comment:
Better to use a more popular json library like jackson or gson (https://github.com/google/gson)